docs/guide/iot/scale-iot-solution-azure.md

```diff
@@ -66,9 +66,8 @@ The requirements for your solution drive the necessary size and number of IoT hu
 
 ### Azure IoT Hub device provisioning service
 
-Azure IoT Hub device provisioning service (DPS) is a helper service for IoT Hub that enables zero-touch, just-in-time provisioning to the right IoT hub without requiring human intervention. It has a soft limit of [10 DPS instances per Azure subscription](/azure/iot-dps/about-iot-dps#quotas-and-limits). You can adjust this limit on a case-by-case basis, but changing the limit might require [proper governance procedures](https://aka.ms/FTAISVGovernance) to be in place for Azure subscription management.
-
-It also has a [soft limit of 1 million registrations](/azure/iot-dps/about-iot-dps#quotas-and-limits) per service instance. Although it's a [soft limit](/azure/azure-resource-manager/management/azure-subscription-service-limits#managing-limits), the service has a hard limit as well. Just like with the IoT hub device limits, you should design with the soft limit as your design limit to avoid issues in the future.
+Azure IoT Hub device provisioning service (DPS) is a helper service for IoT Hub that enables zero-touch, just-in-time provisioning to the right IoT hub without requiring human intervention. It has a hard-limit of [10 DPS instances per Azure subscription](/azure/iot-dps/about-iot-dps#quotas-and-limits). 
+It also has a [hard-limit of 1 million registrations](/azure/iot-dps/about-iot-dps#quotas-and-limits) per service instance. You should design with this limit as your design limit to avoid issues in the future.
```

Comment:
```suggestion
The service also has a [hard-limit of 1 million registrations](/azure/iot-dps/about-iot-dps#quotas-and-limits) per service instance. You must address service limits in your workload design limit to avoid issues in the future.
```